src/google/events/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.events.cloud.audit.v1.events import AuditLogWrittenEvent
+from google.events.cloud.pubsub.v1.events import messagePublishedEvent
```

Comment:
Why is `messagePublishedEvent` not capitalized?

Reply:
This is due to a type in the JSON schemas. It has been fixed.